lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -91,9 +91,11 @@ def request_callback(request: requests.Request):
             agent_logger,
         )
         if access is None:
-            crt.copy_result_tb("token")
+            res = crt.copy_result_tb("token")
         else:
-            crt.copy_result_tb("token", access)
+            res = crt.copy_result_tb("token", access)
+
+        assert res.ok
         # If we got this far without an exception, then the test passes.
```

Comment:
This assertion is better than nothing, but it's a little weak.  The response code should be `201` (right?).  Perhaps it might be `200` (I'm not sure how the test is constructed, exactly), but it shouldn't be anything _else_ in a success case...so, I recommend tightening it.

And, now that `copy_result_tb()` no longer reports errors via exception (and, with the addition of the explicit assertion), this comment should be removed.

Ditto lines 141 and 142.